locale/en.po

```diff
@@ -3308,6 +3311,12 @@ msgstr "Place flagella or slime jets symmetrically at the back of your cell. Oth
 msgid "MICROBE_LOADING_TIP_9"
 msgstr "It matters where you place parts. The shape of your cell impacts your movement and turning speed."
 
+msgid "MICROBE_MEMBRANE_PERCENTAGE_STATISTICS"
+msgstr "  {0}: Found in {1} species"
```

Comment:
It's very undesirable to have padding whitespace directly in the translated text...